l10n/en/products/monitor/breaches.ftl

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/monitor/breaches/
+
+-brand-HIBP = Have I Been Pwned
```

Comment:
Nit: we should only use brand name variables for our own products (third parties should just be a regular string)